quiche/src/lib.rs

```diff
@@ -4799,7 +4799,8 @@ impl Connection {
             is_app_limited: false,
             tx_in_flight: 0,
             lost: 0,
-            has_data,
+            has_data: has_data |
+                (dgram_emitted && path.recovery.gcongestion_enabled()),
```

Comment:
Can you pull this out into a separate variable to make it easier to read.

```
let has_data = if path.recovery.gcongestion_enabled() {
    dgram_emitted || has_data
} else {
    has_data
};
```

Reply:
Done.